lib/commons/aria/arialabel-text.js

```diff
@@ -8,14 +7,11 @@ import { getNodeFromTree } from '../../core/utils';
  * @param {VirtualNode|Element} element
  * @return {string} ARIA label
  */
-function arialabelText(vNode) {
-  if (!(vNode instanceof AbstractVirtualNode)) {
-    if (vNode.nodeType !== 1) {
-      return '';
-    }
-    vNode = getNodeFromTree(vNode);
+export default function arialabelText(element) {
+  const { vNode } = normalizeNode(element);
```

Comment:
Bummer decorators aren't available yet. I'd love to have this work:

```suggestion
@nodeLookup
export default function arialabelText({ vNode }) {
```